Android/app/src/main/java/com/example/yello/common/SecondActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.yello.common;
+
+import android.content.Context;
+import android.support.v4.view.PagerAdapter;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.DatePicker;
+import android.widget.GridView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+
+public class SecondActivity extends AppCompatActivity {
```

Comment:
What does "SecondActivity" mean? Let's choose a better name.

Reply:
I changed it to TrendingClassActivity.
Also, I changed MainActivity to PlaceActivty.